### PR TITLE
image_build: added option to disable KVM

### PR DESCRIPTION
**Pull request by adrianherrera**
This allows images to be built on WSL, etc. Addresses https://github.com/S2E/s2e-env/issues/68

### CODE REVIEW

File: s2e_env/commands/image_build.py [hunk 1]
```diff
@@ -301,12 +308,12 @@ def handle(self, *args, **options):
 
         if options['clean']:
             self._invoke_make(img_build_dir, ['clean'], num_cores)
-            return
+            return None
```
Comment (vitalych) [inline s2e_env/commands/image_build.py]: why?

File: s2e_env/commands/image_build.py [hunk 2]
```diff
@@ -338,7 +345,9 @@ def handle(self, *args, **options):
         _check_product_keys(templates, image_names)
         _check_iso(templates, options['iso_dir'], image_names)
 
-        _check_kvm()
+        if self._use_kvm:
+            _check_kvm()
```
Comment (vitalych) [inline s2e_env/commands/image_build.py]: Does this require fixing guest-images too?

File: s2e_env/commands/image_build.py (hunk 1 above)
Comment (adrianherrera) [reply s2e_env/commands/image_build.py]: Hahah honestly I have no idea why I did this :\ Thanks for pointing it out

File: s2e_env/commands/image_build.py (hunk 2 above)
Comment (adrianherrera) [reply s2e_env/commands/image_build.py]: Hmmm I don't think so? I just "clear" the `QEMU_KVM` variable from [here](https://github.com/S2E/guest-images/blob/master/Makefile.common#L35)

Comment (adrianherrera) [reply s2e_env/commands/image_build.py]: Or am I missing something (or misunderstanding!)

Comment (vitalych) [reply s2e_env/commands/image_build.py]: No, I just forgot how this options was passed, this looks good.

**Comment by vitalych** [comment]
I can't wait to see if this will make image building faster than using nested virtualization.

**Comment by adrianherrera** [comment]
Changes made.

FWIW I don't have a Windows 10 box to test WSL. I just tested on my regular Ubuntu 16.04 box, and while it was substantially slower, it still built an image successfully.

**Pull request merged**